CHANGELOG.md

```diff
@@ -8,6 +8,46 @@
 > requests](https://github.com/fluxcd/toolkit/discussions) for Flux v2
 > and Toolkit components.
 
+## 1.21.2 (2021-02-09)
```

Comment:
This will need an update before the release (NB: release dates are in UTC).

Reply:
Thanks. cef604e5 resolves this